web/codechecker_web/shared/webserver_context.py

```diff
@@ -191,9 +194,12 @@ def severity_map(self):
 def get_context():
     LOG.debug('Loading package config.')
 
-    package_root = os.environ['CC_PACKAGE_ROOT']
+    bin_dir_path = os.environ.get('CC_BIN_DIR', '')
+    lib_dir_path = os.environ.get('CC_LIB_DIR', '')
+    data_files_dir_path = os.environ.get('CC_DATA_FILES_DIR', '')
```

Comment:
Yes, it is a good idea to not have too much parameter for functions when it is possible. For this reason I moved the logic to the __init__ function.

In case of a pypi package the modules, binary files and data files are moved to a different locations based on the operation system: https://docs.python.org/3/install/. And also the user can overwrite the default locations with a prefix option. So it would be really hard to get the other directory paths (e.g.: bin directory) from another file path (e.g.: module directory).